clients/admin-ui/src/features/integrations/configure-monitor/ConfigureMonitorForm.tsx

```diff
@@ -104,15 +135,35 @@ const ConfigureMonitorForm = ({
     ? parseISO(monitor.execution_start_date)
     : Date.now();
 
+  const isLlmClassifierEnabled =
+    monitor?.classify_params?.context_classifier === "llm";
+
   const initialValues = {
     name: monitor?.name ?? "",
     shared_config_id: monitor?.shared_config_id ?? undefined,
     execution_start_date: format(initialDate, "yyyy-MM-dd'T'HH:mm"),
     execution_frequency:
       monitor?.execution_frequency ?? MonitorFrequency.MONTHLY,
+    use_llm_classifier: isLlmClassifierEnabled,
+    model_override: isLlmClassifierEnabled
+      ? (monitor?.classify_params?.model_override ?? undefined)
+      : undefined,
+    prompt_template: isLlmClassifierEnabled
+      ? (monitor?.classify_params?.prompt_template ?? undefined)
+      : undefined,
   };
 
   const frequencyOptions = enumToOptions(MonitorFrequency);
+  const promptTemplateOptions = [
+    {
+      value: ClassifyLlmPromptTemplateOptions.BASE,
+      label: "Base",
```

Comment:
this is just to camelcase the labels in the UI, maybe there's a simpler way to do this...